yabt/buildcontext.py

```diff
@@ -456,19 +456,38 @@ def build(target: Target):
                 built_targets.add(target.name)
                 target.done()
 
-                # TODO: retry flaky tests N times
+                # TODO: attempt flaky tests N times AFTER all the rest passed
                 # TODO: collect stats and print report at the end
                 # TODO: support both "fail fast" (exit on first failure) and
                 #       run all tests (don't exit on first failure) modes
                 target_tested = False
                 if run_tests and 'testable' in target.tags:
                     if self.conf.no_test_cache or not test_cached:
                         logger.info('Testing target {}', target.name)
-                        test_start = time()
-                        self.test_target(target)
-                        target.summary['test_time'] = time() - test_start
-                        logger.info('Test of target {} completed in {} sec',
-                                    target.name, target.summary['test_time'])
+                        attempts = max(target.props.attempts,
```

Comment:
Icky.... default is 1, so the "or" trick doesn't work anymore...

Reply:
how about

```py
attempts = self.conf.test_attempts if target.props.attempts == 1 else target.props.attempts
```

?